consent/manager_test_helpers.go

```diff
@@ -480,17 +480,13 @@ func ManagerTests(deps Deps, m Manager, clientManager client.Manager, fositeMana
 
 					loginVerifier := x.Must(f.ToLoginVerifier(ctx, deps))
 
-					got2, err := m.VerifyAndInvalidateLoginRequest(ctx, f, loginVerifier)
+					got2, err := m.VerifyAndInvalidateLoginRequest(ctx, loginVerifier)
 					require.NoError(t, err)
 					compareAuthenticationRequest(t, c, got2.LoginRequest)
 
-					_, err = m.VerifyAndInvalidateLoginRequest(ctx, nil, loginVerifier)
-					require.Error(t, err)
-
 					loginChallenge = x.Must(f.ToLoginChallenge(ctx, deps))
 					got1, err = m.GetLoginRequest(ctx, loginChallenge)
 					require.NoError(t, err)
-					assert.True(t, got1.WasHandled)
```

Comment:
Should this assertion not still be here? The flow was handled so it should be true right?

Reply:
Previously we persisted the flow in the database, so `VerifyAndInvalidateLoginRequest` set `WasHandled` to `true` in the DB. Since we now store the flow in the login verifier, there is no place to store information about whether the flow was already handled.

This is already true even without this patch. 

If we want double-submit protection, we should build a separate `NonceService` (`new` creates a new nonce, `use` uses a nonce once) to make sure that the verifiers are only used once.

Reply:
I see - do we protect against double submit for the consent verifier? Or can I generate as many authorization codes as I want with the same consent verifier?

Can I log in as often as I want to using the same login verifier?

Reply:
You can log in as often as you want, but the double-submit will be caught at the consent verification step.

I added a test case for consent verifier double submit.